File: app/src/main/java/com/example/recyclerview/Model/Tasks.java
package com.example.recyclerview.Model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Tasks extends RealmObject {

    @PrimaryKey
    private int task_id;
    private String task_name;

    public int getTask_id() {
        return task_id;
    }

    public void setTask_id(int task_id) {
        this.task_id = task_id;
    }

    public String getTask_name() {
        return task_name;
    }

    public void setTask_name(String task_name) {
        this.task_name = task_name;
    }

    public Tasks() {
    }

    public Tasks(String task_name) {

        this.task_name = task_name;
    }
}

File: app/src/main/java/com/example/recyclerview/Model/Tasks.kt
package com.example.recyclerview.Model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Tasks : RealmObject {
    @PrimaryKey
    var task_id = 0
    var task_name: String? = null

    constructor() {}
    constructor(task_name: String?) {
        this.task_name = task_name
    }
}